src/components/Header/header.css

```diff
@@ -0,0 +1,24 @@
+.Header {
+    display: flex;
+    flex-direction: row;
```

Comment:
it should have `flex-direction: row` by defaukt

Reply:
fixed